experiments/integrated/fluxnet/run_fluxnet.jl

```diff
@@ -330,88 +363,105 @@ model_GPP =
         parent(sv.saveval[k].canopy.photosynthesis.GPP)[1] for
         k in 1:length(sv.saveval)
     ] .* 1e6
-
-if drivers.GPP.status == absent
-    plot_daily_avg(
-        "GPP",
-        model_GPP,
-        dt * n,
-        num_days,
-        "μmol/m^2/s",
-        savedir,
-        "Model",
-    )
-else
-    GPP_data =
-        drivers.GPP.values[Int64(t_spinup ÷ DATA_DT):Int64(tf ÷ DATA_DT)] .* 1e6
-    plot_avg_comp(
-        "GPP",
-        model_GPP,
-        dt * n,
-        GPP_data,
-        FT(DATA_DT),
-        num_days,
-        drivers.GPP.units,
-        savedir,
-    )
+if (length(ARGS) > 1) #only plot if a second ARGS "save" is passed when running the code
+    if ARGS[2] == "save"
+        if drivers.GPP.status == absent
+            plot_daily_avg(
+                "GPP",
+                model_GPP,
+                dt * n,
+                num_days,
+                "μmol/m^2/s",
+                savedir,
+                "Model",
+            )
+        else
+            GPP_data =
+                drivers.GPP.values[Int64(t_spinup ÷ DATA_DT):Int64(
+                    tf ÷ DATA_DT,
+                )] .* 1e6
+            plot_avg_comp(
+                "GPP",
+                model_GPP,
+                dt * n,
+                GPP_data,
+                FT(DATA_DT),
+                num_days,
+                drivers.GPP.units,
+                savedir,
+            )
+        end
+    end
 end
 
+
 # SW_OUT
 SW_out_model = [parent(sv.saveval[k].SW_out)[1] for k in 1:length(sv.saveval)]
-if drivers.SW_OUT.status == absent
-    plot_daily_avg(
-        "SW OUT",
-        SW_out_model,
-        dt * n,
-        num_days,
-        "w/m^2",
-        savedir,
-        "model",
-    )
-else
-    SW_out_data = FT.(drivers.SW_OUT.values)[Int64(t_spinup ÷ DATA_DT):Int64(
-        tf ÷ DATA_DT,
-    )]
-    plot_avg_comp(
-        "SW OUT",
-        SW_out_model,
-        dt * n,
-        SW_out_data,
-        FT(DATA_DT),
-        num_days,
-        drivers.SW_OUT.units,
-        savedir,
-    )
+if (length(ARGS) > 1) #only plot if a second ARGS "save" is passed when running the code
```

Comment:
could we wrap the entire plotting section in this if statement? instead of having it for each plot?